apps/devhub/templates/devhub/api/agreement.html

```diff
@@ -0,0 +1,42 @@
+{% extends "devhub/base.html" %}
```

Comment:
Ah right, so here is where I don't understand the UX of the current flow. Here is what happens in our hypothetical flow.

1. User logs into AMO
2. User clicks nav link to generate API key
3. User is redirected to this page
<img width="1243" alt="screen shot 2015-10-05 at 4 02 56 pm" src="https://cloud.githubusercontent.com/assets/2600677/10292281/5cec7428-6b7c-11e5-925b-02ee778993c7.png">
5. User accepts and is sent to actual API page

Now the issue here is that the page shown has nothing to do with API keys. it has a side bar that has a whole bunch of steps, none of which mean anything to someone trying to reach the API page. I must have missed this back in my last commit but this is why I suggested having a separate page for accepting the license.